### DISCUSSION TITLE
Xiaomi sensor: missing battery information in Home assistant

### DISCUSSION THREAD

**Discussion by patwiken** [Koenkk/zigbee2mqtt#13416] [Devices]
Hi, 
Auto discovery is on but some informations are missing in Home Assistant after discovered new devices in Zigbee2MQTT. It happens with the model sensors : WSDCGQ11LM (Aqara temperature/humidity sensor), RTCGQ11LM (Aqara motion sensor).
For Aqara temerature/humidity sensor, only temperature, humidity and pressure informations are integrated, battery level information is missing.
Have you a solution for this problem?

Home Assistant 2021.10.5
Version de Zigbee2MQTT 1.27.0 commit: a9b8808

**Comment by Koenkk** [comment]
`Home Assistant 2021.10.5` -> try updating HA.

**Reply by patwiken** [reply]
Thanks it was the problem but I though it was zigbee2mqtt because this appeared since last updating of it

**Comment by Awebian** [comment]
Same problem, I am already running the latest version of HA 2022.9.1.
Friendly name: Temp Refrigerator
Description: Temperature inside Refrigerator
Last seen: 19 minutes ago
Availability
: Online
Device type: EndDevice
Zigbee Model: lumi.weather
Zigbee Manufacturer: LUMI
Description: Aqara temperature, humidity and pressure sensor
Support status: Supported

IEEE Address: 0x00158d0008651d48
Network address: 0x8951
Firmware build date: 20191205
Firmware version: 3000-0001
Manufacturer: [Xiaomi](https://www.zigbee2mqtt.io/supported-devices/#v=Xiaomi)
Model: [WSDCGQ11LM](https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html#xiaomi-wsdcgq11lm)
Power: Battery ?
Interview completed: True

Log shows:

nfo 2022-09-09 16:49:55MQTT publish: topic 'zigbee2mqtt/Temp Refrigerator', payload '{"battery":null,"humidity":33.05,"last_seen":"2022-09-09T16:49:55-07:00","linkquality":0,"pressure":993.7,"temperature":3.19,"voltage":null}'
